src/bindings/js/node/lib/addon.ts

```diff
@@ -284,8 +284,18 @@ interface Model {
    * It gets all the model outputs as an array
    */
   outputs: Output[];
+  /**
+   * It returns the array of NodeWrap objects.
+   */
+  get_ops(): NodeWrap[];
```

Comment:
At JS side please use `camelCase` for the methods and variables names (but still `snake_case` for c++)